config-linux.md

```diff
@@ -113,6 +116,9 @@ Note that the number of mapping entries MAY be limited by the [kernel][user-name
 
 ## <a name="configLinuxTimeOffset" />Offset for Time Namespace
 
+If the runtime should create an new time namespace for the container, `timeOffsets` should be provided, otherwise, it should not be specified, 
+and it will be ignored by the runtime.
```

Comment:
```suggestion
If the container will join an existing time namespace, `timeOffsets` SHOULD NOT be specified (time namespaces cannot have their mappings changed after being used), and MUST be ignored by the runtime.
Runtimes MAY [generate an error](runtime.md#errors) if the `timeOffsets` is specified and the container is joining an existing time namespace.
```

We can be a bit more strict here because there's no released version of runc with timens support yet.